boards/shields/zest-sensor-imu/zest_sensor_imu.overlay

```diff
@@ -17,4 +20,11 @@
 		compatible = "bosch,bme280";
 		reg = <0x76>;
 	};
-};
\ No newline at end of file
+	
+	iis2dlpc_zest_sensor_imu : iis2dlpc@33 {
+		compatible = "st,iis2dlpc";
+		reg = <0x33>; /* R19 */
```

Comment:
The comment should be removed, as reading the schematics is enough.

It allows not to have to keep this comment updated if we change the name of the resistor.

Reply:
Resolved in : [4dd5c7a](https://github.com/catie-aq/zephyr_zest-sensor-imu/commit/4dd5c7aebb82b512cc8260041e4f3889dd0f67a1).